tools/calendar.md

```diff
@@ -18,6 +18,37 @@ Please make sure, that you have set `Default guest permission under Settings ->
 
 ![Guests can modify events](cal/modify.png)
 
+## Shared calendars
+
+### MayaData shared calendar
+
+MayaData uses shared calendars for events like all hands calls, continuous improvement initiatives & demo days.
+
+A number of shared calendars exist. To add the "MayaData Team Meetings" calendar please do the following:
+
+* Scroll down on the left hand pane of your [Google Calendar](https://calendar.google.com/calendar) and click on the `+` next to `Other calendars`
+
+  <img alt="First step - add shared calendar" src="cal/shared-cal-add-1.png" width="256px">
+
+
+* Choose `From URL` in the dialog:
+ 
+  <img alt="Second step - choose from URL" src="cal/shared-cal-add-2.png" width="256px">
+
+* When presented with the URL you would like to add use:
+
+  ```
+  https://calendar.google.com/calendar/u/0?cid=Y19zMmNlYzZwOGdhaXUxNmRjaDJ0YnFmazZ1a0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t
```

Comment:
Shouldn't we add the Meet Mayadata as well? Culture repo is public and it might help promotion. Also clicking on the link is enough (at least as I've tested it in FF) so the process of adding calendar by URL is not really needed. I will do that.

Reply:
Added.